src/components/uiOptions/js/UIOptions.js

```diff
@@ -26,4 +26,38 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
         ]
     });
 
+    /** A configuration of UIOptions which is suitable for statically localised contexts. It accepts two top-level
+     * options,
+     * {String} locale - The initial locale in which UIOptions should render
+     * {String} direction - A suitable value for the `dir` attribute of the UIOptions container - this may take
+     * values `ltr`, `rtl` or `auto`
+     */
+    fluid.defaults("fluid.uiOptions.multilingual", {
+        gradeNames: ["fluid.uiOptions"],
+        distributeOptions: {
+            source: "{that}.options.locale",
+            target: "{that messageLoader}.options.resourceOptions.locale"
+        },
```

Comment:
This isn't really needed, as `prefsEditorLoader` options block can already be used to feed in these values and distribute them to the correct locations. See: https://github.com/fluid-project/trivet/pull/148#discussion_r652754785